apps/demo/src/index.tsx

```diff
@@ -1,13 +1,16 @@
-import { StrictMode } from 'react';
-import ReactDOM from 'react-dom';
-
 import './styles.css'; // global styles
 
+import { assertNonNull } from '@h5web/app';
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+
 import DemoApp from './DemoApp';
 
-ReactDOM.render(
+const rootElem = document.querySelector('#root');
+assertNonNull(rootElem);
```

Comment:
`rootElem` can be `null`... React recommends doing `createRoot(rootElem!)` but this gives an ESLint warning. I think an assertion is cleaner. I re-exported the one we have in both `lib` and `app` to avoid duplication.